utilities/tests/general-helper-functions-test.metta

```diff
@@ -38,3 +38,59 @@
 !(assertEqual (findAndReplace Z 1 (B C A (A B C) F G A)) (B C A (A B C) F G A))
 !(assertEqual (findAndReplace A Z (A C A (A B C) A G A)) (Z C Z (A B C) Z G Z))
 !(assertEqual (findAndReplace F (A B C) (B C A (A B C) F G A)) (B C A (A B C) (A B C) G A))
+
+;; Tests for until function
+
+;; These are functions used for testing until function.
+
+;; It returns True only if the input is zero.
+(: isZero (-> Number Bool))
+(= (isZero $x) (== $x 0))
+
+;; It will return True if greater than 10.
+(: greaterThanTen (-> Number Bool))
+(= (greaterThanTen $x) (> $x 10))
+
```

Comment:
why do you need this function? I mean it is a trivial check and very specific

Reply:
Yeah it is. I only need it to for testing the `until` function, hence the reason I put it in the test file.